databuilder/task/neo4j_staleness_removal_task.py

```diff
@@ -53,8 +55,9 @@ def get_scope(self):
 
     def init(self, conf):
         # type: (ConfigTree) -> None
-
-        conf = conf.with_fallback(DEFAULT_CONFIG)
+        conf = Scoped.get_scoped_conf(conf, self.get_scope())\
```

Comment:
so it won't have deep copy issue when doing this? I thought the deepcopy happens in `with_fallback` call?

Reply:
Correct. The config will have only on current scope so as long as current scope does not have non-copiable object, it won't fail which we can control when configure.

I first tried not using with_fallback, but just use merge config but it introduced different failure, and at this point this should work.